com.github.dawidd6.checksumo.yml

```
@@ -0,0 +1,22 @@
+app-id: com.github.dawidd6.checksumo
+runtime: org.gnome.Platform
+runtime-version: '3.38'
+sdk: org.gnome.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.golang
+command: checksumo
+finish-args:
+  - --socket=x11
```

```suggestion
  - --socket=fallback-x11
```
This will select wayland by default if available, and also includes the --share=ipc permission which is otherwise required for x11 to be performant.